OpenRA.Game/TextNotificationsManager.cs

```diff
@@ -77,6 +87,8 @@ static bool IsPoolEnabled(TextNotificationPool pool)
 
 			return pool == TextNotificationPool.Chat ||
 				pool == TextNotificationPool.System ||
+				pool == TextNotificationPool.Join ||
+				pool == TextNotificationPool.Leave ||
```

Comment:
When I wrote this initially I wanted it to be explicit about what pools are enabled (even though they always are) and to anticipate future pools (see https://github.com/OpenRA/OpenRA/pull/18905#discussion_r649496801). Now that the number of pools has grows this is starting to look ugly so it can be refactored.